lib/components/ContentModal/ContentModal.css

```diff
@@ -0,0 +1,101 @@
+.overlay {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background-color: rgba(var(--vds-color-slate-darker), 0.5);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 1000;
```

Comment:
it would be better to create a separate file with a list of z-indices we use and include it here as a variable

Reply:
done